tests/cloudferrylib/os/compute/test_nova.py

```diff
@@ -13,6 +13,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+# pylint: disable=protected-access
```

Comment:
Let's keep pylint warning, it reminds that tests are written incorrectly and require refactoring